lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -228,7 +228,7 @@ describe('CategoryRenderer', () => {
 
     const bestPracticeCat = sampleResults.categories['best-practices'];
     const categoryDOM2 = renderer.render(bestPracticeCat, sampleResults.categoryGroups);
-    assert.ok(!categoryDOM2.querySelector('.lh-clump--notapplicable'));
```

Comment:
what happened here?

Reply:
This test is looking for not applicable audits & originally the sample results didn't have any N/A audits in the `best-practices` category. When I added `preload-fonts` to `best-practices` & didn't update the `sample_v2.json`, `preload-fonts` was N/A so this test failed

This should be reverted to its original state once I update `sample_v2.json` so that `preload_fonts` isn't NA